source/modules/src/main/sql/database-modules/grid/01-types.sql

```diff
@@ -0,0 +1,7 @@
+/*
+ * hexagon_type
+ * ------------
+ * The type of a hexagon, which is used for statistics of a calculation for example.
+ */
+CREATE TYPE grid.hexagon_type AS ENUM
+	('relevant_hexagons', 'exceeding_hexagons', 'above_cl_hexagons');
```

Comment:
Now that I see this, I'm not sure if we should include this in database-modules or keep it in Calculator. I've already changed this set in a feature branch in any case, so it would have to be updated if we do decide to keep it in here AND use it in calculator.